test/e2e/storage/drivers/csi.go

```diff
@@ -857,8 +857,8 @@ func (g *gcePDCSIDriver) PrepareTest(f *framework.Framework) (*storageframework.
 		Framework: f,
 	}
 
-	if framework.ProviderIs("gke") {
-		framework.Logf("The csi gce-pd driver is automatically installed in GKE. Skipping driver installation.")
+	if framework.ProviderIs("gce", "gke") {
```

Comment:
The only k/k jobs that run these tests are using kube-up. Other platforms that may be running these tests and were depending on the driver being installed as part of the test will need to deploy the driver using their own cluster deployment mechanism.

Reply:
I wonder if we should also just remove the whole section here that installs the driver. If we keep the specs here in the test config, we'll have to continue maintaining the images here.

Also, if we can assume the test never installs the driver in all cases, then we can remove the "serial" tag from these tests and start running them in parallel, which should greatly help speed up our optional storage pull jobs.

Reply:
That's a good idea. Maybe best to defer to another PR to keep this one focused?

Reply:
Followup pr is fine.

FYI we already skip gce pd tests if the provider is not gce or gke: https://github.com/kubernetes/kubernetes/blob/427994f76d2f8401eb0a2522915b7719ae876f96/test/e2e/storage/drivers/csi.go#L823